portality/lib/formulaic.py

```diff
@@ -910,11 +911,14 @@ def draft(self, *args, **kwargs):
     def validate(self):
         self.pre_validate()
         f = self.form
-        valid = False
         if f is not None:
-            valid = f.validate()
+            if f.application_status.data in self._status_not_requiring_validation:
+                return True
+            else:
+                return f.validate()
```

Comment:
This shouldn't be in the formulaic library file.  Instead override the validation method in the application_processor module for the form instances where it is relevant.  In this case, I'd copy the style ManEdJournalReview.validate method to the AdminApplicationReview class, and make it aware that if the form.application_status contains `rejected` then it shouldn't validate.

Reply:
The error is that I do not check whether the `f.application_status` exists at all. I would just add that check and leave it here. If any form gives the user possibility to change the status to "rejected" it should also not require validation in such case. And it might be the case that the scope of responsibility will be changed one day.